lib/components/List/listItemLabel.module.css

```diff
@@ -49,6 +49,10 @@
   margin: 0;
 }
 
```

Comment:
Foreslår å sette en eksplisitt font-weight også for bold for å få lettere fleksibilitet rundt bytting av default label.